app/src/main/java/de/tum/in/tumcampusapp/auxiliary/calendar/IntegratedCalendarEvent.java

```diff
@@ -77,19 +78,19 @@ private static String getEventLocationFromCursor(Cursor cEvents) {
         return eventLocation.trim();
     }
 
-    private static long getEventIdFromCursor(Cursor cEvents) {
-        return cEvents.getLong(0);
+    private static long getEventIdFromCursor(CalendarItem cEvents) {
+        return Long.parseLong(cEvents.getNr());
     }
 
-    private static Calendar getEventEndFromCursor(Cursor cEvents) {
-        String eventEnd = cEvents.getString(6);
+    private static Calendar getEventEndFromCursor(CalendarItem cEvents) {
+        String eventEnd = cEvents.getDtend();
         Calendar result = Calendar.getInstance();
         result.setTime(Utils.getDateTime(eventEnd));
         return result;
     }
 
-    private static Calendar getEventStartFromCursor(Cursor cEvents) {
-        String eventStart = cEvents.getString(5);
+    private static Calendar getEventStartFromCursor(CalendarItem cEvents) {
```

Comment:
The `*FromCursor()` methods don't make any sense anymore, since they don't take a Cursor...
This should be refactored to instance methods of `CalendarItem`